presentation-api/index.html

```diff
@@ -41,122 +37,97 @@ <h3>Background</h3>
     <option value="cherries">
     <option value="strawberry">
   </datalist>
-  <button id="sendFruit" disabled>presentationConnection.send(message)</button>
+  <button id="sendMessage">presentationConnection.send(message)</button>
 </p>
 <p>
-  <input id="message" type="text" placeholder="Send a message!">
-  <button id="sendMessage" disabled>presentationConnection.send(message)</button>
+  <button id="close">presentationConnection.close()</button>
+  <button id="terminate">presentationConnection.terminate()</button>
 </p>
 <p>
-  <input id="presentationId" type="text" placeholder="presentationId">
-  <button id="reconnect" disabled>presentationRequest.reconnect(presentationId)</button>
+  <input id="presentationId" type="text" placeholder="Enter a presentation ID...">
+  <button id="reconnect">presentationRequest.reconnect(presentationId)</button>
 </p>
-<p>
-  <button id="close" disabled>presentationConnection.close()</button>
-  <button id="terminate" disabled>presentationConnection.terminate()</button>
-</p>
-{% endcapture %}
 {% include output_helper.html initial_output_content=initial_output_content %}
 
+<script>
+  log = ChromeSamples.log;
+</script>
+
 {% capture js %}
-const presentationUrls = ['receiver/index.html'];
+const presentationRequest = new PresentationRequest(['receiver/index.html']);
```

Comment:
Can you set navigator.presentation.defaultRequest here, so this presentation is the default choice when casting from the toolbar/menu?

Reply:
Done.